terraform/aws/modules/efs/efs.tf

```diff
@@ -0,0 +1,11 @@
+/*This will create EFS in bursting mode.The mode can be changed as per the requirement*/
+resource "aws_efs_file_system" "elastic_file_system" {
+  creation_token = var.creation_token
+
+  lifecycle_policy {
+    transition_to_ia = "AFTER_30_DAYS"
+  }
```

Comment:
pls add options to set encryption true/false as needed and the kms_key_id as well.

Reply:
@qburst-praven This is done.